test/numbers.jl

```diff
@@ -1,5 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+≣(a, b) = isequal(a, b) # convenient for comparing NaNs
```

Comment:
Noted. The reason I defined as a function was to avoid something like #18598 again (in the case someone defines `≣` as another function in another test).